packages/mobile/src/web3/saga.ts

```diff
@@ -133,9 +122,15 @@ function* checkSyncProgressWorker() {
           continue
         }
 
-        Logger.debug(TAG, 'Sync Progress Completed')
-        yield put(setSyncProgress(100))
-        yield put(setIsReady(true))
+        const latestBlock: Block = yield getLatestBlock()
+        if (latestBlock && latestBlock.number > 0) {
+          yield put(setLatestBlockNumber(latestBlock.number)) // Don't understand why we need this
```

Comment:
I couldn't figure out why we're doing this as I couldn't find an usage of this store variable that it's set here.

Reply:
For the debug screen I guess? Probably doesn't need to be in store though, you're right.